draft-ietf-tls-tls13.md

```diff
@@ -2204,7 +2186,9 @@ signature
   The values indicate anonymous signatures, RSASSA-PKCS1-v1_5
   {{RFC3447}}, DSA {{DSS}}, and ECDSA {{ECDSA}}, respectively.  The
   "anonymous" value is meaningless in this context but used in
-  {{server-key-share}}.  It MUST NOT appear in this extension.
+  {{server-key-share}}. It MUST NOT appear in this extension.
+  The use of DSA is deprecated. The dsa_RESERVED value MUST NOT 
+  be offered or negotiated by any implementation.
```

Comment:
Actually, disregard my previous note on this part of the text. Don't improve the anon bit; remove it. It appears to be obsolete, and this value isn't used anymore. The Server Key Message in that section is fundamentally different now. Change the "anonymous" value in the enum to "anon_RESERVED" and change this text to something like this, instead:
> The use of DSA and the anonymous value are deprecated.
> The anon_RESERVED and dsa_RESERVED values MUST
> NOT be offered or negotiated by any implementation.


>   anonymous_RESERVED placeholder value is meaningless in this context but used in
>  {{server-key-share}}.  It MUST NOT appear in this extension.
+  {{server-key-share}}. It MUST NOT appear in this extension.
+  The use of DSA is deprecated. The dsa_RESERVED value MUST NOT 
+  be offered or negotiated by any implementation.